app/test_split_integer.py

```diff
@@ -2,20 +2,33 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+        sum(split_integer(10, 5)) == 10
+    ), "Sum of the parts should be equal to value!"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (
+        split_integer(12, 3).count(12 / 3) == 3
+    ), "Should split into equal parts when value divisible by parts!"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    result_list = split_integer(2, 1)
+    assert (
+        len(result_list) == 1 and result_list[0] == 2
+    ), "Should return part equals to value when split into one part!"
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    result_list = split_integer(13, 7)
+    sorted_result_list = split_integer(13, 7)
```

Comment:
Are you sure you are actually checking if the list is sorted? In my opinion, there is no need to create separate variables here. Item number one in the checklist